cli/src/pcluster/config/cluster_config.py

```diff
@@ -113,10 +113,36 @@ def __init__(
         self,
         size: int = None,
         encrypted: bool = None,
+        volume_type: str = None,
+        iops: int = None,
+        throughput: int = None,
     ):
         super().__init__()
         self.size = Resource.init_param(size, default=EBS_VOLUME_SIZE_DEFAULT)
         self.encrypted = Resource.init_param(encrypted, default=True)
+        self.volume_type = Resource.init_param(volume_type, default=EBS_VOLUME_TYPE_DEFAULT)
+        self.iops = Resource.init_param(iops, default=EBS_VOLUME_TYPE_IOPS_DEFAULT.get(self.volume_type))
+        self.throughput = Resource.init_param(throughput, default=125 if self.volume_type == "gp3" else None)
```

Comment:
we might create a mapping like `EBS_VOLUME_TYPE_IOPS_DEFAULT` for throughput too or an utilitty function to retrieve the value from this map or return None if not specified, rather than having this case here.